ostelco-ios-client/Assets/Assets.xcassets/instructionsESimContinue.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "screenshot1.png",
```

Comment:
Can you please export images from Zeplin as PDF rather than 3 pngs? If you take a look at, for example `newWindowIcon`, you can see how to set up a single universal vector PDF, which saves space both at install time and in the repo.

Reply:
Will do!

Reply:
@designatednerd are we going to use gifs / mp4 at a later point for these images?

The two first images are available as pdf, but the two last are not according to zeplin
`This asset contains a ‘bitmap’ image inside. So no vector assets is generated.`

Just wondering, before I bug our designer.

Reply:
Re: gifs - The designs all appear to be static images. 

Re: vectors - yeah I saw that on a couple things yesterday as well. I would say leave notes in the Zeplin document and see if there's a way to get that output as a vector image. You don't need to wait for him to finish that to merge this though.